Available Apps.md

```
@@ -21,7 +23,11 @@
 
 ### Available Apps
 
-| Name                       | Stack        | Hafs                                         | Warsh                                         | Page Image | Transparent PNG | SVG | English UI | Arabic UI |
-|----------------------------|--------------|--------------------------------------------- | --------------------------------------------- | ---------- | --------------- | --- | ---------- | --------- |
-| Open Mushaf                | React Native |https://qurancomplex.gov.sa/en/techquran/dev/ | https://qurancomplex.gov.sa/en/techquran/dev/ | Yes        | No              | No  | Yes        | Yes       |
-| Quran Androind (Quran.com) | Kotlin       |                                              |                                               |            |                 |     |            |           |
\ No newline at end of file
+| Name                                                               | Stack        | Hafs                                          | Warsh                                         | Page Image | Transparent PNG | SVG | English UI | Arabic UI |
+| ------------------------------------------------------------------ | ------------ | --------------------------------------------- | --------------------------------------------- | ---------- | --------------- | --- | ---------- | --------- |
+| Open Mushaf                                                        | React Native | https://qurancomplex.gov.sa/en/techquran/dev/ | https://qurancomplex.gov.sa/en/techquran/dev/ | Yes        | No              | No  | Yes        | Yes       |
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)